db/schema.rb

```diff
@@ -46,6 +46,23 @@
     t.index ["state_id"], name: "index_cities_on_state_id"
   end
 
+  create_table "clients", id: :serial, force: :cascade do |t|
+    t.string "name"
+    t.integer "company_id"
+    t.datetime "created_at", precision: nil, null: false
+    t.datetime "updated_at", precision: nil, null: false
+    t.boolean "active", default: true
+    t.index ["company_id"], name: "index_clients_on_company_id"
+  end
+
+  create_table "companies", id: :serial, force: :cascade do |t|
+    t.string "name"
+    t.datetime "created_at", precision: nil
+    t.datetime "updated_at", precision: nil
+    t.string "avatar"
+    t.integer "end_period"
+  end
```

Comment:
Is is related to `pr` state? Wondering if it came from another issue you're working on.

Reply:
Oh, this wasn't supposed to be there.